src/components/layout/DefaultLayout.tsx

```diff
@@ -97,22 +99,90 @@ const DefaultLayout: React.FC<{
                 .filter(nav => nav.block === 'header')
                 .map(nav =>
                   nav.external ? (
-                    <a key={nav.label} href={nav.href} target="_blank" rel="noopener noreferrer">
-                      <StyledNavLinkButton type="link">{nav.label}</StyledNavLinkButton>
-                    </a>
+                    <Menu>
+                      <MenuButton
+                        onClick={() => nav.href && window.open(nav.href, '_blank', 'noopener=yes,noreferrer=yes')}
+                      >
+                        <StyledNavButton bg="#fff">{nav.label}</StyledNavButton>
+                      </MenuButton>
```

Comment:
NITS: can just combine [`as` props](https://chakra-ui.com/guides/as-prop) and styled-component here
```tsx
<StyledMenuButton as="a" target="_blank" rel="noopener noreferrer" href={nav.href}>
  {nav.label}
</StyleMenuButton>

```